app/move/app/assign/controllers/base.js

```diff
@@ -33,7 +33,11 @@ class AssignBaseController extends CreateBaseController {
   }
 
   setMoveSummary(req, res, next) {
-    res.locals.moveSummary = presenters.moveToMetaListComponent(req.move)
+    const sessionData = req.sessionModel.toJSON()
```

Comment:
From the tests below I can see we're using the session of profile and to location. Would that be impacted if a user has opened more than one move at a time? While things stored on the req are bound to the individual req context, the session is shared so is there a chance of something getting overridden?

Reply:
Good question, but no, that sessionModel is also bound to the req for that form wizard. Since we introduced individual form wizards for moves to help with new tab behaviour this won't have an impact.

I'm actually trying to move the new/edit base controllers away from using another "model" concept on top of the existing form wizard `sessionModel` concept.

Basically we added a whole other layer on top of something that already existed which I don't think we need anymore.